fury/data/fetcher.py

```diff
@@ -267,6 +273,54 @@ def fetcher():
     return fetcher
 
 
+def fetch_viz_gltf(name=None, mode='glTF'):
+    """Downloads glTF samples from Khronos Group Github.
+
+    Parameter
+    -----------
+    name: str, list, optional
+        Name of the glTF model (for e.g. Box, BoxTextured, FlightHelmet, etc)
+        https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0
+        Default: None, Downloads essential glTF samples for tests.
+
+    mode: str, optional
+        Type of glTF format.
+        You can choose from different options (e.g. glTF, glTF-Embedded, etc)
+        Default: glTF, `.bin` and texture files are stored separately.
+    """
+    if name is None:
+        name = ['BoxTextured', 'Duck', 'CesiumMilkTruck', 'CesiumMan']
+
+    if type(name) == list:
+        for element in name:
+            fetch_viz_gltf(element)
+    else:
+        url = GITHUB_API_URL + name + '/' + mode
+        try:
+            request = urlopen(url).read()
+            request = json.loads(request)
+        except HTTPError as e:
+            print("Model {} does not exist, Please check name and mode"
+                  .format(name))
+            return None
+
+        name = pjoin(* ['glTF', name, mode])
+        folder = pjoin(fury_home, name)
+
+        if not os.path.exists(folder):
+            print("Creating new folder ", folder)
+            os.makedirs(folder)
+
+        for file in request:
+            download_url = file['download_url']
+            filename = download_url.split('/')[-1]
+            fullpath = os.path.join(folder, filename)
+
+            if not os.path.exists(fullpath):
+                print('Downloading file: {}'.format(filename))
+                urlretrieve(download_url, filename=fullpath)
```

Comment:
as we said, let's use asyncio and ahttpio

Reply:
YeAH, that needs to be implemented I am working on it, after that this PR will be ready to be merged.